### PR TITLE
CORE-8121 Update REST API doc

### PR DESCRIPTION
**Pull request by davidcurrie**

### CODE REVIEW

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 1]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
 The Corda REST API enables operators to interact with a running version of Corda.
 To access and invoke the REST API:
 
-1. To access the RPC endpoint of a Corda cluster, forward the port by running the following command in a terminal window:
+1. If the REST API has not been exposed externally to the cluster via a load balancer, forward the port by running the following command in a terminal window:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:14]:
```suggestion
1. If the REST API has not been exposed externally to the cluster via a load balancer, forward the port by running the following command in a terminal window, replacing `<NAMESPACE>` with the Kubernetes namespace that Corda is installed in:
```

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 2]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
...
 
    ```sh
-   kubectl port-forward -n corda deploy/corda-rpc-worker 8888
+   kubectl port-forward -n <NAMESPACE> deploy/corda-rpc-worker 8888
    ```
 
-   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+   Where `<NAMESPACE>` is the Kubernetes namespace that Corda is installed in.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:20]:
```suggestion

```

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 3]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
...
 
-   {{< note >}}
-   The RPC endpoint is protected by a self-signed certificate.
-   {{< /note >}}
+2. If you did not explicitly specify the username for the initial admin user at install time, the default is `admin`.
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:22]:
```suggestion
   If you did not explicitly specify the username for the initial admin user at install time, the default is `admin`.
```

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 4]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
...
+
+3. If you did not explicitly specify the password for the initial admin user at install time, the generated value can be retrieved as follows:
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:24]:
```suggestion
If you did not explicitly specify the password for the initial admin user at install time, the generated value can be retrieved as follows:
```

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 5]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
...
    ```
 
-   You can then access the Swagger documentation for the RPC endpoint at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
+   Where `<NAMESPACE>` is the Kubernetes namespace that Corda is installed in.
 
-   {{< note >}}
-   The RPC endpoint is protected by a self-signed certificate.
-   {{< /note >}}
+2. If you did not explicitly specify the username for the initial admin user at install time, the default is `admin`.
+
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:23]:
```suggestion
```

File: content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md [hunk 6]
```diff
@@ -11,22 +11,35 @@ section_menu: corda-5-alpha
...
+   USERNAME=admin
+   PASSWORD=$(kubectl get secret -n <NAMESPACE> corda-initial-admin-user -o go-template="{{ .data.password | base64decode }}")
+   curl -u $USERNAME:$PASSWORD -k $API_ENDPOINT/api/v1/hello
    ```
 
+   {{< note >}}
+   The REST API is protected by a self-signed certificate.
+   {{< /note >}}
+
+5. You can access the Swagger documentation for the REST API at the path `/api/1/swagger`. For example, when using port forwarding the documentation is available at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-alpha/developing/rest-api/rest-api.md:43]:
```suggestion
You can access the Swagger documentation for the REST API at the path `/api/1/swagger`. For example, when using port forwarding the documentation is available at [https://localhost:8888/api/v1/swagger](https://localhost:8888/api/v1/swagger).
```

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
Thx @nadinequinr3

**Pull request merged**